Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py

```diff
@@ -910,17 +910,15 @@ def main():  # pragma: no cover
             human_readable, outputs, response = batch_update_findings_command(client, args)
         elif command == 'fetch-incidents':
             fetch_incidents(client)
-            sys.exit(0)
+            return
```

Comment:
`return` is for linter?

Reply:
no, it's to avoid the rest of the function